src/edfi-performance-test/edfi_performance_test/tasks/pipeclean/education.py

```diff
@@ -37,10 +37,7 @@ class EducationOrganizationNetworkAssociationPipecleanTest(EdFiPipecleanTestBase
 
 class EducationOrganizationPeerAssociationPipecleanTest(EdFiPipecleanTestBase):
     def _touch_put_endpoint(self, resource_id, default_attributes):
-        # This resource has no non-FK data attributes, so we'll update one of those
-        default_attributes["educationOrganizationReference"][
-            "educationOrganizationId"
-        ] = SchoolClient.shared_elementary_school_id()
+        # This resource has no non-key attributes, so there is nothing to update
```

Comment:
ODS/API interestingly does not invalidate a PUT request that tries to change a natural key - instead it just ignores those changes. Meadowlark invalidates the request and responds with 400.